### PR TITLE
Document issue in Preview 2 with SocketsHttpHandler on single core machines

### PR DESCRIPTION
**Pull request by rmkerr**
Documenting corefx issue [#28979](https://github.com/dotnet/corefx/issues/28979). The instructions on switching to the platform handler are taken from the [Preview 2 release blog post](https://blogs.msdn.microsoft.com/dotnet/2018/04/11/announcing-net-core-2-1-preview-2/).

### CODE REVIEW

File: release-notes/2.1/Preview/2.1.0-preview2-known-issues.md [hunk 1]
```diff
@@ -26,4 +26,24 @@ We have temporarily removed the Preview 2 installers from the Linux package feed
...
 sudo apt install dotnet-host=2.0.6-1
-```
\ No newline at end of file
+```
+
+## SocketsHttpHandler hangs on single core machines
+
+There is an issue with the new default HttpClientHandler that will cause hangs on single core machines when multiple connections are opened simultaneously. [corefx/issues/28979](https://github.com/dotnet/corefx/issues/28979). This issue has since been fixed, but is present in the Preview 2 build.
+
+**Workaround** - Disable SocketsHttpHandler as the default, and fall back to the platform handler.
```
Comment (karelz) [inline release-notes/2.1/Preview/2.1.0-preview2-known-issues.md]: Let's reword it to "Use 2+ core machines, or use older HttpClientHandler (as described in [Preview 2 blog post](https://blogs.msdn.microsoft.com/dotnet/2018/04/11/announcing-net-core-2-1-preview-2/))"

Comment (rmkerr) [reply release-notes/2.1/Preview/2.1.0-preview2-known-issues.md]: Updated this in the latest commit.
